src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -613,6 +613,75 @@ public void testLastIndexOfLastBoundaries() {
         Assert.assertEquals(result, expected);
     }
 
+    @Test
+    public void testLastIndexOfOneMismatchLastBoundaries() {
+        final String reference = "AAAACCCCTTTTGGGG";
+
+        // match right boundary of reference
+        String query = "TGAGG";
+        int result = Utils.lastIndexOfOneMismatch(reference.getBytes(), query.getBytes());
+        int expected = 11;
+        Assert.assertEquals(result, expected);
+
+        // match left boundary of reference
+        query = "AAGAC";
+        result = Utils.lastIndexOfOneMismatch(reference.getBytes(), query.getBytes());
+        expected = 0;
```

Comment:
Inline this variable.  BTW, if you have never done so in the IDE before, use IntelliJ to do this (right click --> refactor --> inline).